src/utils/colonyMotions.ts

```diff
@@ -93,3 +94,15 @@ export const MOTION_TAG_MAP = {
     name: MSG.invalidTag,
   },
 };
+
+export const getMotionRequiredStake = (
```

Comment:
Make sure to replace this in the other places in the dapp where we use a required stake